src/model.ts

```diff
@@ -84,6 +84,7 @@ export interface ICreateJobModel {
   environment: string;
   parameters?: IJobParameter[];
   outputFormats?: IOutputFormat[];
+  computeType?: string;
```

Comment:
How about the rest of the model properties, for example `idempotency_token`, `tags` etc.?

Reply:
I was planning to add those when I added the UI controls/display for them.